### PR TITLE
Add support for Arizona Public Service

### PR DESCRIPTION
**Pull request by seferino-fernandez**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -83,4 +83,5 @@ max-complexity = 25
 dev = [
     "pytest>=7,<8",
     "python-dotenv>=1,<2",
+    "cryptography>=43.0.0"
```
Comment (tronikos) [inline pyproject.toml:86]: You added this to the wrong section. Move it to line 15, keep them in alphabetical order.

File: src/opower/utilities/aps.py [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+                0
+            ][
+                "sASPDetails"
+            ][
+                0
+            ][
+                "sAID"
+            ]
+
+        ACCOUNT_SERVICE_ID = f"{account_id}_{service_address_id}"
```
Comment (tronikos) [inline src/opower/utilities/aps.py:95]: Use all caps only for constants

**Review by tronikos — CHANGES_REQUESTED** [review]

File: pyproject.toml (hunk 1 above)
Comment (seferino-fernandez) [reply pyproject.toml]: Oops, fixed!

File: src/opower/utilities/aps.py (hunk 2 above)
Comment (seferino-fernandez) [reply src/opower/utilities/aps.py]: Updated!

File: pyproject.toml [hunk 3]
```diff
@@ -13,6 +13,7 @@ dependencies = [
     "aiozoneinfo>=0.1",
     "arrow>=1.2",
     "pyotp>=2.0",
+    "cryptography>=43.0.0",
```
Comment (tronikos) [inline pyproject.toml:16]: Move it one line above so that this list is ordered.

**Review by tronikos — CHANGES_REQUESTED** [review]
Install and run pre-commit. See readme

File: pyproject.toml (hunk 3 above)
Comment (seferino-fernandez) [reply pyproject.toml:15]: Done!

**Comment by tronikos** [comment]
I'm bumping the version in https://github.com/home-assistant/core/pull/125981 and mentioning the support in the documentation in https://github.com/home-assistant/home-assistant.io/pull/34739

Could you create a virtual integration in Home Assistant? You need 3 PRs similar to https://github.com/tronikos/opower/pull/87#issuecomment-2253017806

**Comment by seferino-fernandez** [comment]
Here are my PR's to the Home Assistant project.
Virtual Integration: https://github.com/home-assistant/core/pull/126014
Documentation: https://github.com/home-assistant/home-assistant.io/pull/34749
Brands: https://github.com/home-assistant/brands/pull/5864

**Pull request merged**